gradle.properties

```diff
@@ -1,3 +1,3 @@
-micronautVersion=2.0.0
-libVersion=1.2.1
-testcontainersVersion=1.15.2
+micronautVersion=3.5.5
+libVersion=2.0.0
```

Comment:
shall we simply set the libVersion to 3.5 to reflect that this is compatible with micronaut 3.5 ?

Reply:
well then ;)